source/jormungandr/jormungandr/new_relic.py

```diff
@@ -94,3 +95,33 @@ def ignore():
         except:
             logger = logging.getLogger(__name__)
             logger.exception('failure while ignoring transaction')
+
+
+class DistributedEvent:
+    """
+    Custom event that we publish to New Relic for distributed scenario
+    """
+
+    def __init__(self, service, call_name, group_name):
+        self.event_params = {
+            "service": type(service).__name__,
```

Comment:
minor (same): simple quotes if possible

I let you look (ctrl+F) along this PR if you want to change it.